src/networking.c

```diff
@@ -2774,6 +2775,11 @@ NULL
                         addReplyErrorObject(c,shared.syntaxerr);
                         return;
                     }
+
+                    if (c->argc == 4) {
+                        addReplyError(c, "The skipme parameter cannot be used alone");
+                        return;
+                    }
```

Comment:
Are we sure we want this restriction, it can be considered either a breaking change or a bug fix.
@madolson WDYT?
If we're go for this, then lets add a comment here https://github.com/redis/redis-doc/blob/8c32f75c8a341bae8565b50b91c29467ceafd616/commands/client-kill.md?plain=1#L19 too.

Reply:
For what it's worth I have called "CLIENT KILL SKIPME YES", to kill everyone else. I'm also not a big fan of introducing a breaking change unless there is a compelling reason?

Reply:
Makes sense. So I'm for keeping `CLIENT KILL SKIPME YES`.

Reply:
oh so this is the expected result (killing all the clients, including or not including `me`)? 
Somehow I feel this combination is a bit dangerous. If it is its own intention, then yes make scene

So far, skipme by default is "YES", so kind of like"CLIENT KILL" will equal "CLIENT KILL SKIPME YES",
except for the first on, syntaxerr will be reported. 

If so, i will revert this part, i think it can be recorded on the document, providing a path to kill all clients

Reply:
Yeah, the behavior is to kill everyone including yourself. It's a bit weird, but so it goes.